package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
- [ ] When running the command `npm run start` the project doesn’t open in the browser. You miss some scripts in your `package.json` file, please kindly make sure you setup your project correctly 🙃 🙃

Your `scripts` should be similar to the example below 👇 👇

```json
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```

Reply:
Kindly, I suggest you review this topic carefully, your app does not start with the traditional "npm run start" command, check the configuration of your package.json file or replace it with a correct one.